### ISSUE TITLE
WMS GetFeatureInfo

### ISSUE THREAD

**Issue by zoran995** [TerriaJS/terriajs#4431]
For WMS 1.3.0 layer the GetFeatureInfo request will fail with error 
`<ServiceException code="InvalidPoint"> I and J incorrectly specified </ServiceException>`
this is only the case when GetFeatureInfo request is sent to 1.3.0 version of service.

According the OGC specification for WMS 1.3.0 the GetFeatureInfo specifies I and J as parameters for GetFeatureInfo, while in the specification for WMS 1.1.1 parameters were X and Y.

Are we generating this request or is it done on the cesium end?

Steps to reproduce 
* Add the WMS 1.3.0 service to the map (for example this [one](https://wms.inspire.geoportail.lu/geoserver/au/wms?SERVICE=WMS&service=WMS&version=1.3.0&request=GetCapabilities) - luxembourg)
* try to get feature info for some unit
* observer the network request sent

Incorect request sent
```
https://wms.inspire.geoportail.lu/geoserver/au/wms?SERVICE=WMS&
version=1.3.0&
service=WMS&
request=GetFeatureInfo&
layers=AU.AdministrativeUnit&
bbox=626172.1357121654%2C6261721.357121639%2C782715.1696402058%2C6418264.391049679&
width=256&
height=256&
crs=EPSG%3A3857&
query_layers=AU.AdministrativeUnit&
x=98&
y=29&
info_format=application%2Fjson
```

Correct request sent

```
https://wms.inspire.geoportail.lu/geoserver/au/wms?SERVICE=WMS&
version=1.3.0&
service=WMS&
request=GetFeatureInfo&
layers=AU.AdministrativeUnit&
bbox=626172.1357121654%2C6261721.357121639%2C782715.1696402058%2C6418264.391049679&
width=256&
height=256&
crs=EPSG%3A3857&
query_layers=AU.AdministrativeUnit&
i=98&
j=29&
info_format=application%2Fjson

**Comment by zoran995** [comment]
This is resolved on Cesium end (https://github.com/CesiumGS/cesium/pull/9143) and included in the latest release (1.74). Will close this once terriajs-cesium is updated.

**Comment by nf-s** [comment]
I believe this is working now, thanks @zoran995 !
